### ISSUE TITLE
Better Way to Get Person's Job Title and Company 

### ISSUE THREAD

**Issue by Turreted** [joeyism/linkedin_scraper#46]
I'm currently using regex to extract information from the person.experiences field

```python
user_company_info = re.findall(
    r"b\'(.+?)\'", str(person.experiences[0])
)
```

However, this method is unreliable. Since this is a fairly important feature, I was wondering if linkedin_scraper had a builtin method to handle this. If not, I'd be happy to work on its implementation.

**Comment by joeyism** [comment]
I don't think there's currently one. If you have time, you can work on it, submit and merge request, and I'll take a look

**Comment by Turreted** [comment]
Would it be better if the person's most recent company and job title were class attributes, or if each person has a list of previous jobs and titles, with the item at index 0 being their most recent?

**Comment by Turreted** [comment]
Solution is at request #48

**Comment by joeyism** [comment]
Thanks for the MR. It's been released in [2.5.0](https://github.com/joeyism/linkedin_scraper/releases/tag/2.5.0)
